cartographer_ros/src/sensor_bridge.cc

```diff
@@ -41,7 +41,11 @@ void SensorBridge::AddOdometryMessage(const string& topic,
 void SensorBridge::AddImuMessage(const string& topic,
                                  const sensor_msgs::Imu::ConstPtr& msg) {
   auto sensor_data = ::cartographer::common::make_unique<SensorData>(
-      msg->header.frame_id, ToCartographer(*msg));
+      msg->header.frame_id,
```

Comment:
Should there be a CHECK, that the covariances do not start with -1, i.e. data is there?